packages/rainbow-sprinkles/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "rainbow-sprinkles",
-  "version": "0.14.2",
+  "version": "0.14.3",
```

Comment:
We can leave this out, changesets handles versioning for us (which happens in a separate PR)

```suggestion
  "version": "0.14.2",
```